tests/reg_issue4750/reg_issue4750.fz

```diff
@@ -0,0 +1,28 @@
+# This file is part of the Fuzion language implementation.
+#
+# The Fuzion language implementation is free software: you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as published
+# by the Free Software Foundation, version 3 of the License.
+#
+# The Fuzion language implementation is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public
+# License for more details.
+#
+# You should have received a copy of the GNU General Public License along with The
+# Fuzion language implementation.  If not, see <https://www.gnu.org/licenses/>.
+
+
+# -----------------------------------------------------------------------
+#
+#  Tokiwa Software GmbH, Germany
+#
+#  Source code of Fuzion test reg_issue4750
+#
+# -----------------------------------------------------------------------
+
+reg_issue4750 =>
+
+  say ([(3,4),(5,6)].map (.1) |> .sum)
+
+  say <| [("Error:","This does"),("not","work.")].map (.1)
```

Comment:
can you add test cases that go further and to things like `.map (.1.0)`, `.map (.1[i])`, `.map (.1.as_string)`, etc.?